systemvm/patches/debian/vpn/etc/ipsec.d/l2tp.conf

```diff
@@ -30,4 +35,5 @@ conn L2TP-PSK
         # ----------------------------------------------------------
         # Change 'ignore' to 'add' to enable this configuration.
         #
+        rightsubnetwithin=10.1.2.0/8
```

Comment:
@pdion891 @remibergsma @jayapalu what is the best configuration to be used here; or we allow admins to pass in a value we override here?